doc/lua_api.txt

```diff
@@ -5380,20 +5380,49 @@ Definition tables
             place = <SimpleSoundSpec>,
             place_failed = <SimpleSoundSpec>,
         },
-        drop = "",
+
+        drop = "", --[[
         ^ Name of dropped node when dug. Default is the node itself.
-        ^ Alternatively:
+          Using a table allows multiple items, random selection among items,
+          palette control and tool filtering:
         drop = {
             max_items = 1,  -- Maximum number of items to drop.
             items = {  -- Choose max_items randomly from this list.
```

Comment:
What list? The list of items seems to come way after this comment, so there's some space for confusion.